cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file should be deleted or edited to make it specific to your project.

Reply:
Thanks.